jib-core/src/test/java/com/google/cloud/tools/jib/cache/LayerEntriesSelectorTest.java

```diff
@@ -63,6 +67,29 @@
         .collect(ImmutableList.toImmutableList());
   }
 
+  @Test
+  public void testLayerEntryTemplate_fileModified() throws IOException {
+    TemporaryFolder temporaryFolder = new TemporaryFolder();
+    temporaryFolder.create();
+    Path layerFile = temporaryFolder.newFolder("testFolder").toPath().resolve("file");
```

Comment:
This file should probably exist - we wouldn't ever be adding a layer entry for a file that doesn't exist.